src/main/java/uk/gov/companieshouse/web/accounts/controller/smallfull/IntangibleAmortisationPolicyController.java

```diff
@@ -0,0 +1,92 @@
+public class IntangibleAmortisationPolicyController extends BaseController {
+
+    private static final String INTANGIBLE_AMORTISATION_POLICY = "intangibleAmortisationPolicy";
+
+    @Autowired
+    private IntangibleAmortisationPolicyService intangibleAmortisationPolicyService;
+
+    @GetMapping
+    public String getIntangibleAmortisationPolicy(@PathVariable String companyNumber,
+                                                  @PathVariable String transactionId,
+                                                  @PathVariable String companyAccountsId,
+                                                  Model model,
+                                                  HttpServletRequest request) {
+
+        addBackPageAttributeToModel(model, companyNumber, transactionId, companyAccountsId);
+
+        try {
+            model.addAttribute(INTANGIBLE_AMORTISATION_POLICY,
+                    intangibleAmortisationPolicyService.getIntangibleAmortisationPolicy(transactionId, companyAccountsId));
+        } catch (ServiceException e) {
+
+            LOGGER.errorRequest(request, e.getMessage(), e);
+            return ERROR_VIEW;
+        }
+
+        return getTemplateName();
+    }
+
+    @PostMapping
+    public String submitIntangibleAmortisationPolicy(@PathVariable String companyNumber,
+                                                     @PathVariable String transactionId,
+                                                     @PathVariable String companyAccountsId,
+                                                     @ModelAttribute(INTANGIBLE_AMORTISATION_POLICY) @Valid IntangibleAmortisationPolicy intangibleAmortisationPolicy,
+                                                     BindingResult bindingResult,
+                                                     Model model,
+                                                     HttpServletRequest request) {
+
+        addBackPageAttributeToModel(model, companyNumber, transactionId, companyAccountsId);
+
+        if (bindingResult.hasErrors()) {
+            return getTemplateName();
+        }
+
+        try {
+            List<ValidationError> validationErrors = intangibleAmortisationPolicyService
+                    .submitIntangibleAmortisationPolicy(transactionId, companyAccountsId, intangibleAmortisationPolicy);
+
+            if (!validationErrors.isEmpty()) {
+                bindValidationErrors(bindingResult, validationErrors);
+                return getTemplateName();
+            }
+        } catch (ServiceException e) {
+
+            LOGGER.errorRequest(request, e.getMessage(), e);
+            return ERROR_VIEW;
+        }
+
+        return Navigator.getNextControllerRedirect(this.getClass(), companyNumber, transactionId, companyAccountsId);
+    }
+
+    @Override
+    protected String getTemplateName() {
+        return "smallfull/intangibleAmortisationPolicy";
```

Comment:
possibly make this a static final variable?

Reply:
I don't think it wins us much, it's another variable assignment for no gain but I'm happy to be guided here if it's a preference?